crates/primitives/src/hex_literal.rs

```diff
@@ -94,6 +94,8 @@ macro_rules! hex {
         $crate::hex_literal::decode::<{ $crate::hex_literal::len(STRINGS) }>(STRINGS)
     }};
 }
+#[doc(hidden)] // Use `crate::hex` directly instead!
+pub use crate::hex;
```

Comment:
unfortunately this will also re-export the `hex` crate through this path, so I added `doc(hidden)` to limit the impact